app/assets/stylesheets/pages/_spots.scss

```diff
@@ -5,6 +5,15 @@
 .spots__message-wrapper {
   @include wrapper();
   @include spacer();
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.spots__message {
+  text-align: center;
+  margin: $spacing-5 0px;
```

Comment:
If you don't need to set all values, consider not using the shorthand notations. (See: https://codeguide.co/#css-shorthand).

So instead of `margin: $spacing-5 0px`,

```
margin-top: $spacing-5;
margin-bottom: $spacing-5;
```